HoloStorageConnector/README.md

```diff
@@ -89,4 +88,8 @@ This package also provided some prefabs to save your development time, which cou
     <img src="../HoloRepositoryDemoApplication/Images/DemoScene.png" height="400">
 </p>
 
-This package also provided a Demo scene to guide the developer how to use this package, which you can find [here](../HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Demo).
\ No newline at end of file
+This package also provided a Demo scene to guide the developer how to use this package, which you can find [here](../HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Demo).
+
+## Technologies
+The following technologies are used in this component:
```

Comment:
Good idea to list technologies, but for the sake of completeness, maybe also mention Unity, MRTK, ..